.github/workflows/watch-dependencies.yaml

```diff
@@ -149,12 +152,16 @@ jobs:
 
       - name: Get latest version of jupyterhub
         id: latest
+        shell: python
         run: |
-          latest_version=$(
-              curl -s https://pypi.org/pypi/jupyterhub/json \
-            | jq -r .info.version
-          )
-          echo "::set-output name=version::$latest_version"
+          import requests
+
+          request = requests.get("https://pypi.org/pypi/jupyterhub/json")
+          data = request.json()
+          releases = data["releases"]
+          latest_version = list(releases.keys())[-1]
```

Comment:
I'm not sure the order of this dict is guaranteed, or what its order is, if it is stable (e.g. it would be wrong when we do a backport release if it's by time). I think you need to sort by parsed version: `sorted(releases.keys(), key=packaging.version.Version)[-1]`.